bin/deploy

```diff
@@ -112,10 +110,23 @@ test -f "$LSSTSW/miniconda/.packages.deployed" || ( # conda packages
     fi
 
     if [[ $BLEED_DEPLOY == true ]]; then
-        # The conda Intel MKL linked packages are intentionally avoided.
-        # See: https://jira.lsstcorp.org/browse/DM-5105
-        CONDA_BLEED_PACKAGES=(nomkl numpy scipy matplotlib requests cython sqlalchemy astropy pandas future)
+        # lsst_distrib / lsst_sims deps
+        CONDA_BLEED_PACKAGES=(numpy scipy matplotlib requests cython sqlalchemy astropy pandas future)
         ARGS+=("${CONDA_BLEED_PACKAGES[@]}")
+
+        # matplotlib pulls in qt and there are concerns about qt5 causing
+        # breakage, so we are pinning to 4.x
+        ARGS+=("pyqt==4.11.4")
+
+        # The conda Intel MKL linked packages are intentionally avoided [on
+        # linux].
+        # See: https://jira.lsstcorp.org/browse/DM-5105
+        if [[ $(uname -s) == Linux* ]]; then
+            ARGS+=("nomkl")
+        fi
+
+        # lsst_build requires pyyaml
+        ARGS+=("pyyaml")
```

Comment:
If you are being pedantic about listing `lsst_build` dependencies I think it needs the `future` package and in theory you can remove `future` from the list of products needed by `lsst_apps/distrib` above because we ship our own EUPS future.